Updating SDP-K1 PinNames.h.

malavikasajikumar opened the pull request:
### Description

<!--
    Required
    Add here detailed changes summary, testing results, dependencies
    Good example: https://os.mbed.com/docs/mbed-os/latest/contributing/workflow.html (Pull request template)
-->
- Removing LED pin definitions based on color.
- Removing and updating incorrect USB pin definitions.
- Adding comments for Arduino SPI and I2C pins.

### Pull request type

<!--
    Required
    Please add only one X to one of the following types. Do not fill multiple types (split the pull request otherwise).
    Please note this is not a GitHub task list, indenting the boxes or changing the format to add a '.' or '*' in front
    of them would change the meaning incorrectly. The only changes to be made are to add a description text under the
    description heading and to add a 'x' to the correct box.
-->
    [x] Fix
    [ ] Refactor
    [ ] Target update
    [ ] Functionality change
    [ ] Docs update
    [ ] Test update
    [ ] Breaking change

### Reviewers

<!--
    Optional
    Request additional reviewers with @username
-->
@maclobdell @0xc0170 
### Release Notes

<!--
    Optional
    In case of breaking changes, functionality changes or refactors, please add release notes here. 
    For more information, please see [the contributing guidelines](https://os.mbed.com/docs/mbed-os/latest/contributing/workflow.html#pull-request-types).
-->

0xc0170 commented on targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F469xI/TARGET_SDP_K1/PinNames.h:
```diff
@@ -310,9 +310,9 @@ typedef enum {
-    LED_RED     = LED1,
-    LED_ORANGE  = LED2,
-    LED_GREEN   = LED3,
+					   
```
Can you remove pls dead spaces?

0xc0170 replied:
One more, the commit msg should contain details from the comment here, please add why you are updating the file.

Could be:

```
SDP-K1: Updating pin names

Removing LED pin definitions based on color.
Removing and updating incorrect USB pin definitions.
Adding comments for Arduino SPI and I2C pins.
```

0xc0170 reviewed (changes requested):
After spaces are removed, LGTM

malavikasajikumar committed:
Updating SDP-K1 PinNames.h file.

- Removing LED pin definitions based on color.
- Removing and updating incorrect USB pin definitions.
- Adding comments for Arduino SPI and I2C pins.
